astroquery/simbad/core.py

```diff
@@ -323,10 +316,7 @@ def list_wildcards(self):
         [abc] : Exactly one character taken in the list.
                 Can also be defined by a range of characters: [A-Z]
         """
-        for key in self._ORDERED_WILDCARDS:
-            print("{key} : {value}\n".format(key=key,
-                                             value=self.WILDCARDS[key]))
-        return
+        print("\n\n".join(f"{k} : {v}" for k, v in self.WILDCARDS.items()))
```

Comment:
it's ok to keep it like this, but overall I value non-single-letter variables even when the intention is clear like in this line. (comment is for here and also for cases below)